aquarius/events/processors.py

```diff
@@ -326,3 +326,10 @@ def check_update(self, new_asset, old_asset, sender_address):
             return False
 
         return True
+
+class MetadataStateProcessor(EventProcessor):
+    def make_record(self,data):
```

Comment:
On Hold*: I think you don't need to inherit from EventProcessor. There are no common applications e.g. RBAC permission checks, etc. And you will need a different (smaller) set of arguments. I think just the metadata state value, token address, chain id and es_instance are enough. I've added a similar processor in v4main which is already merged.

(*) the only case where I could see us needing all the arguments (and the EventProcessor inheritance) is if we can have Metadata State change from something else TO 0 => we need to re-add the asset back into Aquarius, and then we have to delegate to the EventMetadataCreated event. But I'm not sure if that is a possibility. I'll ask.